### PR TITLE
Update red_teaming.json

### PR DESCRIPTION
**Pull request by WulfPlasma**
Added King Phisher and Gophish

### CODE REVIEW

**Comment by WulfPlasma** [comment]
@noraj lmk if everything looks good and you're ready to merge.

File: data/tools/red_teaming.json [hunk 1]
```diff
@@ -63,6 +63,22 @@
       "language": "Shell",
       "price": "Free",
       "online": "False"
-    }
+    },
+    {
+      "name": "King Phisher",
```
Comment (noraj) [inline data/tools/red_teaming.json:68]: You have to sort them alphabetically. see [Format conventions](https://inventory.raw.pm/docs/docs/en/format/) & [Adding/changing content](https://inventory.raw.pm/docs/docs/en/add/#addingchanging-content)

File: data/tools/red_teaming.json [hunk 2]
```diff
@@ -63,6 +63,22 @@
...
+      "online": "False"
+    },
+    {
+      "name": "Gophish",
+      "source": "https://getgophish.com/",
+      "description": "Gophish is an open-source phishing toolkit designed for businesses and penetration testers. It provides the ability to quickly and easily setup and execute phishing engagements and security awareness training.",
+      "language": "Go",
+      "price": "Free",
+      "online": "True"
+    },
```
Comment (noraj) [inline data/tools/red_teaming.json:82]: extra comma

File: data/tools/red_teaming.json [hunk 3]
```diff
@@ -63,6 +63,22 @@
...
+      "name": "King Phisher",
+      "source": "https://github.com/rsmusllp/king-phisher",
+      "description": "A tool for testing and promoting user awareness by simulating real world phishing attacks",
+      "language": "Python",
+      "price": "Free",
+      "online": "False"
+    },
+    {
+      "name": "Gophish",
+      "source": "https://getgophish.com/",
```
Comment (noraj) [inline data/tools/red_teaming.json:77]: this is the website, not the source repo

File: data/tools/red_teaming.json [hunk 4]
```diff
@@ -63,6 +63,22 @@
...
+      "description": "Gophish is an open-source phishing toolkit designed for businesses and penetration testers. It provides the ability to quickly and easily setup and execute phishing engagements and security awareness training.",
```
Comment (noraj) [inline data/tools/red_teaming.json:78]: Format conventions of teh description does not contains tool name, etc.
Do not only copy paste the disclaimer of the tool.

**Review by noraj — CHANGES_REQUESTED** [review]
I fixed all that in 5020dedef07ef51b154419f9ea4339134dbd4999 for this time, so you don't need to update it.
Thanks for your contribution.

**Comment by WulfPlasma** [comment]
Thank you for letting me work on this, I’ll keep everything in mind that was fixed if I contribute again.

**Comment by WulfPlasma** [comment]
@noraj does it still count as invalid even though it was merged?

**Comment by noraj** [comment]
Yeah I tagged it as invalid because I had too much change to do to make it follow [Format conventions](https://inventory.raw.pm/docs/docs/en/format/) & [Adding/changing content](https://inventory.raw.pm/docs/docs/en/add/#addingchanging-content).
Please follow those guidelines for your next contributions :smiley:

**Pull request merged**